doc.go

```diff
@@ -18,7 +18,7 @@ limitations under the License.
 // declarative Kubernetes operators to manage the lifecycle of a
 // deployment.
 //
-// Getting Started
+// # Getting Started
```

Comment:
Is it paragraph markup on markdown format?

Reply:
This has to do with `gofmt` updates related to bumping the Go version to Go 1.19.

If you take a look at the **Headings** section of this documentation https://tip.golang.org/doc/comment you can see that as of Go 1.19 the new heading format is similar to Markdown and uses the `#` prefix:

> The # syntax was added in Go 1.19. Before Go 1.19, headings were identified implicitly by single-line paragraphs satisfying certain conditions, most notably the lack of any terminating punctuation.

> Gofmt reformats [lines treated as implicit headings](https://github.com/golang/proposal/blob/master/design/51082-godocfmt.md#headings) by earlier versions of Go to use # headings instead. If the reformatting is not appropriate—that is, if the line was not meant to be a heading—the easiest way to make it a paragraph is to introduce terminating punctuation such as a period or colon, or to break it into two lines. 

I hope this helps!

Reply:
Thank you for detailed and helpful comment.
I understood that. 👍